Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clean.py

```diff
@@ -81,6 +82,9 @@ def cleanup(cls, repository, argument, remote_target=None):
 
             argument = pr.head
 
+        push_env = os.environ.copy()
+        push_env['VERBOSITY'] = str(verbosity)
+
```

Comment:
The proliferation of this boilerplate makes me wonder if it'd be easier to handle this in `webkitcorepy.run`, somehow. If we were using the `logging` module, we could set VERBOSITY based on the current log level.

Not something that needs to be addressed, just something I noticed!

Reply:
If we had more subprocess scripts that did this sort of thing, I would be inclined to pass this in `webkitcorepy.run`, for now, it's just this one...if we do it again, we'll definitely put this in `webkitcorepy.run`